### PR TITLE
fixed issue #130, so people can select from browse by agency multiple time

### LINKED ISSUE #130: Results from previous agency search do not clear upon search for new agency
When you search for an agency on the home page https://code.gov/, and then go back to the home page to search by another agency, the results from the previous agency are still there, until you clear them by clicking the X. Not the user experience I would expect.

**Describe the bug**

**To Reproduce**

1. Click on browse by agency
2. Select an agency
3. Click on code.gov to go home
4. click on browse by agency again
5. Select another agency
Results for both agencies are there. You have to x out the first agency to get those search results to go away

**Expected behavior**

Would expect results for one agency to go away once I go back home and select another agency. 

**Screenshots**

**Desktop (please complete the following information):**
Chrome 

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by DanielJDufour**
A similar PR may already be submitted!
Please search among the [Pull request](../) before creating one.

Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:

For more information, see the `CONTRIBUTING` guide.

**Summary**

This PR fixes/implements the following **bugs/features**

* [ ] Don't accumulate federal agency filters when you select browse by agency multiple times

Explain the **motivation** for making this change. What existing problem does the pull request solve?
Issue #130 

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.
Manually tested and also added to pre-deploy testing checklist

**Code formatting**

**Closing issues**

Fixes #130

### CODE REVIEW

**Comment by DanielJDufour** [comment]
Don't review this yet. Just discovered a bug when I select `All` in Browse by Agency

**Comment by DanielJDufour** [comment]
Ready for review now

File: src/components/home-banner/home-banner.container.js [hunk 1]
```diff
@@ -25,21 +25,21 @@ const mapDispatchToProps = dispatch => {
       const value = normalize(event.target.value)
       if (value !== 'browse by agency') {
         let url
```
Comment (ftonato) [inline src/components/home-banner/home-banner.container.js]:
```javascript
let url = `/browse-projects?agencies=${value}&page=1&size=10&sort=data_quality`
```

File: src/components/home-banner/home-banner.container.js [hunk 2]
```diff
@@ -25,21 +25,21 @@ const mapDispatchToProps = dispatch => {
...
+        const params = {
+          page: 1,
+          size: 10,
+          sort: 'data_quality',
+          filters: []
+        }
         if (value === 'all') {
           url = '/browse-projects?page=1&size=10&sort=data_quality'
```
Comment (ftonato) [inline src/components/home-banner/home-banner.container.js]:
```javascript
url = url.split(`agencies=${value}&`).join('')
```

**Comment by ftonato** [comment]
Hello @DanielJDufour,

Your changes that enough, but I suggest simple updates (you can see above).
If you believe is interesting, accept, otherwise just accept this _PR_ ;)

**Pull request merged**